packages/data-explorer-ui/src/components/common/Code/code.styles.ts

```diff
@@ -0,0 +1,14 @@
+import styled from "@emotion/styled";
+
+/**
+ * Basic code block
+ */
+export const CodeBlock = styled("pre")`
```

Comment:
Nice work @Ajalle ! 👏🏽 

As per discussion in slack - and after thinking this through a little bit more - let's update with the following changes.

The <code> element has font-family of “monospace” which is overriding the font-family Roboto Mono specified on the pre element. This got me thinking about potential flexibility of the use of code and would love you to update the ticket to do the following:
1) Add to MuiCssBaseline in packages/data-explorer-ui/src/theme/common/components.ts the following:
      code: {
        ...theme.typography[TEXT_BODY_400_2_LINES],
        fontFamily: "Roboto Mono, monospace",
        fontSize: 12,
      },
2) Add to MuiCssBaseline the required pre styles from your Code component such as margin, work-break and white-space.
3) Remove the now deprecated styles from CodeBlock. You can either leave the CodeBlock defined as pre or just replace in the component directly with pre.